app/facades/menus/index_facade.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Menus
+  class IndexFacade
+    attr_reader :menus
+
+    def initialize
```

Comment:
should be `menus` method. You don't pass any params, so no need to have initialize